core/base_service.go

```diff
@@ -627,7 +627,11 @@ func getErrorMessage(responseMap map[string]interface{}, statusCode int) string
 		var errors Errors
 		responseBuffer, _ := json.Marshal(responseMap)
 		if err := json.Unmarshal(responseBuffer, &errors); err == nil {
-			return errors.Errors[0].Message
+			if len(errors.Errors) > 0 {
```

Comment:
Wow, how did we exist for so long with this bug in place?  LOL
This must be the first time that a service returned an error response with an explicit `null` value for "errors".

Please check the other cores to make sure we don't have a similar bug there.